tests/python/test_assert.py

```diff
@@ -107,6 +107,19 @@ def func():
     func()
 
 
+@test_utils.test(require=ti.extension.assertion,
+                 debug=True,
+                 check_out_of_bound=True,
+                 gdb_trigger=False)
+def test_assert_with_check_oob():
```

Comment:
What is the difference between this test and `test_assert_ok` above? IIRC `check_out_of_bound` is implicitly true when `debug=True`.

Reply:
@strongoier this one is added since it's the exact repro which errored out before so I guess it's just to make sure it disappeared.